src/EventSourcing/Infrastructure/Projection/ProjectionTable.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EventSourcingWorkshop\EventSourcing\Infrastructure\Projection;
+
+/**
+ * Represents a projection table to be modified (both DDL and DML) at runtime.
```

Comment:
Possibly worth declaring that this is not an EventSourcing concept, but just an abstraction on top of `doctrine/dbal`.

May even get rid of it, and allow the projector to use `Doctrine\DBAL\Connection` on each event, although it allows for a lot of misuse :thinking:

Reply:
DDL is not part of this anymore.

Reply:
DDL is present with `truncate()` - keeping this.

Reply:
98b3a599bba2bee89b256655a34edaa4c6a5d67d